site/class-1-load/Senpaizuri/master.css

```diff
@@ -0,0 +1,58 @@
+*,*:before,*:after{
```

Comment:
Could you rename this file to `index.css`? (it’s in the [assignment](https://github.com/cmda-fe3/course-17-18/blob/master/class-1.md#-step-h)).